lib/ramble/ramble/appkit.py

```diff
@@ -24,5 +23,9 @@
 import ramble.language.application_language
 from ramble.language.application_language import *
 from ramble.language.shared_language import *
+from ramble.util.logger import logger
+
+# Import new logger as tty to preserve old behavior
+from ramble.util.logger import logger as tty
```

Comment:
Yes, exactly.

And yes, there are trade-offs.

Using `llnl.util.tty` would print to stdout instead of the experiment log file, while this allows the application.py to print to it's experiment log file instead.

The primary downside is that the full interface of `llnl.util.tty` is not implemented in `ramble.util.logger` so it's possible some symbols will be missed if they are necessary.